src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/dft.hpp

```diff
@@ -4,21 +4,21 @@
 
 #pragma once
 
-#include <tuple>
 #include <string>
+#include <tuple>
 
-#include "shared_test_classes/base/layer_test_utils.hpp"
 #include "ngraph_functions/builders.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
 
 namespace LayerTestsDefinitions {
 
-typedef std::tuple<
-        InferenceEngine::SizeVector, // Input shapes
-        InferenceEngine::Precision,  // Input precision
-        std::vector<int64_t>,  // Axes
-        std::vector<int64_t>,  // Signal size
-        ngraph::helpers::DFTOpType,
-        std::string> DFTParams;   // Device name
+typedef std::tuple<InferenceEngine::SizeVector,  // Input shapes
+                   InferenceEngine::Precision,   // Input precision
+                   std::vector<int64_t>,         // Axes
+                   std::vector<int64_t>,         // Signal size
+                   ngraph::helpers::DFTOpType,
+                   std::string>
+    DFTParams;  // Device name
```

Comment:
Seems unncessary changes. Please do not change common code unless it is needed.

Reply:
reverted